neurox/interpretation/gaussian_probe.py

```diff
@@ -0,0 +1,282 @@
+"""Module for Gaussian Method to rank neurons
+
+This module implements the Gaussian Method to rank the neuron importance
+
+.. seealso::
+        Lucas Torroba Hennigen, Adina Williams, and Ryan Cotterell. Intrinsic probing through dimension
+selection. In Proceedings of the 2020 Conference on Empirical Methods in Natural Language
+Processing (EMNLP), pp. 197–216, Online, 2020. Association for Computational Linguistics. doi:
+10.18653/v1/2020.emnlp-main.15.a
+
+"""
+import numpy as np
+
+import torch
+
+import torch.distributions.multivariate_normal as mn
+
+from . import metrics
+
+
+class GaussianProbe:
+    def __init__(self, X, y):
```

Comment:
Can you add a comment here pointing to the original repo (where we have borrowed the code from). Something like: "Modified from original version at <link>"